charts/k8s-infra/templates/apikey-self-telemetry-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if and (not .Values.presets.selfTelemetry.apiKeyExistingSecretName) .Values.presets.selfTelemetry.signozApiKey }}
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: {{ include "k8s-infra.fullname" . }}-self-telemetry-apikey-secret
+  labels:
+    {{- include "k8s-infra.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": "pre-install,pre-upgrade"
+    "helm.sh/hook-delete-policy": "before-hook-creation"
+data:
+  signoz-apikey: {{ .Values.signozApiKey | b64enc | quote }}
```

Comment:
```suggestion
  signoz-apikey: {{ .Values.presets.selfTelemetry.signozApiKey | b64enc | quote }}
```

Reply:
can you also write a test where OTLP exporter is enabled and a different SigNoz API Key is set for that?

Reply:
thanks, done